doc/en/reference/reference.rst

```diff
@@ -1819,6 +1819,19 @@ passed multiple times. The expected format is ``name=value``. For example::
             clean_db
 
 
+.. confval:: verbosity_assertions
```

Comment:
Two reasons:

* My use case is to always enable `-vv` for assertions without increasing the verbosity of the rest of the output. Setting that in the configuration file allows this to be enabled for a project without being explicitly added each time `pytest` is run.
* My initial implementation took an arbitrary string when adding a new level. The thought was that any plugin might add their own type. This would mean that there could potentially be a large number of command line options related to fine-grained verbosity, which seemed harder to manage. With the move to `pytest` statically defining each type, that will keep the N small, so I'm less apposed to the idea.
* With the existence of `addopts` and `--override-ini`, it isn't clear if there are rules about what should be a command line option version a configuration file setting.

Reply:
I agree this is something one might want to configure mainly via the configuration file, so a configuration value seems reasonable.